src/main/java/uk/gov/ons/census/fwmt/jobservice/message/GatewayActionsReceiver.java

```diff
@@ -76,4 +56,36 @@ private void processMessage(String message) throws GatewayException {
     }
     return dto;
   }
+
+  private void checkForRequestType(String actualMessage) throws GatewayException, IOException {
+    JsonNode actualMessageRootNode = jsonObjectMapper.readTree(actualMessage);
```

Comment:
Im guessing  the IOException is coming from the objectMapper, wrap it with a try/catch and throw the gateway Exception